crates/agent/src/logs.rs

```diff
@@ -102,7 +115,7 @@ pub async fn serve_sink(
             trace!(%token, %stream, %line, "rx (cont)");
             tokens.push(token);
             streams.push(stream);
-            lines.push(line);
+            lines.push(sanitize_null_bytes(line));
```

Comment:
also need to get the line that's pop'd in the select above.

Reply:
Whoops, good catch!